Resources/Locale/en-US/cuffs/components/handcuff-component.ftl

```diff
@@ -1,7 +1,7 @@
 handcuff-component-target-self = You start restraining yourself.
 handcuff-component-cuffs-broken-error = The restraints are broken!
-handcuff-component-target-has-no-hands-error = {$targetName} has no hands!
-handcuff-component-target-has-no-free-hands-error = {$targetName} has no free hands!
+handcuff-component-target-has-no-hands-error = {$targetName} { CONJUGATE-HAVE($targetEnt) } no hands!
+handcuff-component-target-has-no-free-hands-error = {$targetName} { CONJUGATE-HAVE($targetEnt) } no free hands!
```

Comment:
"have" is wrong to be used here

"Myra have no free hands"

"Myra has no free hands"